packages/perseus/src/widgets/radio/radio.tsx

```diff
@@ -334,6 +350,7 @@ class Radio extends React.Component<Props> {
      * Turn on rationale display for the currently selected choices. Note that
      * this leaves rationales on for choices that are already showing
      * rationales.
+     * @deprecated Internal only. Use `showSolutions` prop instead.
```

Comment:
Thanks for deprecating this.  I hadn't ever looked too closely at this function, but I really don't like it. Calling an imperative method which takes it's prop value and mutates it and then tells its parent to re-render it.

Reply:
After I make similar changes to use the prop elsewhere we can deprecate it on the renderer as well